src/ec/packages/ec-component-contextual-navigation/__snapshots__/contextual-navigation.test.js.snap

```diff
@@ -0,0 +1,324 @@
+    >
+      <button
+        class="ecl-button ecl-button--ghost ecl-contextual-navigation__more"
+        data-ecl-contextual-navigation-more="true"
+        type="submit"
+      >
+        <span
+          class="ecl-button__container"
+        >
+          <span
+            class="ecl-button__label"
+            data-ecl-label=""
+          >
+            More
+          </span>
+          <svg
+            aria-hidden="true"
+            class="ecl-icon ecl-icon--fluid ecl-icon--rotate-90 ecl-button__icon ecl-button__icon--after"
+            focusable="false"
+          >
+            <use
+              xlink:href="static/icons#ui--corner-arrow"
+            />
+          </svg>
+        </span>
+      </button>
+    </li>
+  </ul>
+</nav>
+`;
+
+exports[`EC - Contextual Navigation renders correctly with extra class names 1`] = `
+<nav
+  class="ecl-contextual-navigation custom-class-1 custom-class-2"
+  data-ecl-contextual-navigation=""
+>
+  <div
+    class="ecl-contextual-navigation__label"
+  >
+    Label for contextual navigation
+  </div>
+  <ul
+    class="ecl-contextual-navigation__list"
+    data-ecl-contextual-navigation-list=""
+  >
+    <li
+      class="ecl-contextual-navigation__item"
+    >
+      <a
+        class="ecl-link ecl-link--standalone ecl-contextual-navigation__link"
+        href="/example"
+      >
+        Item 1
+      </a>
+    </li>
+    <li
+      class="ecl-contextual-navigation__item"
+    >
+      <a
+        class="ecl-link ecl-link--standalone ecl-contextual-navigation__link"
+        href="/example"
+      >
+        Item 2
+      </a>
+    </li>
+    <li
+      class="ecl-contextual-navigation__item"
+    >
+      <a
+        class="ecl-link ecl-link--standalone ecl-contextual-navigation__link"
+        href="/example"
+      >
+        Item 3
+      </a>
+    </li>
+    <li
+      class="ecl-contextual-navigation__item ecl-contextual-navigation__item--collapsed"
+    >
+      <a
+        class="ecl-link ecl-link--standalone ecl-contextual-navigation__link"
+        href="/example"
+      >
+        Item 4
+      </a>
+    </li>
+    <li
+      class="ecl-contextual-navigation__item ecl-contextual-navigation__item--collapsed"
+    >
+      <a
+        class="ecl-link ecl-link--standalone ecl-contextual-navigation__link"
+        href="/example"
+      >
+        Item 5
+      </a>
+    </li>
+    <li
+      class="ecl-contextual-navigation__item ecl-contextual-navigation__item--collapsed"
+    >
+      <a
+        class="ecl-link ecl-link--standalone ecl-contextual-navigation__link"
+        href="/example"
+      >
+        Item 6
+      </a>
+    </li>
+    <li
+      class="ecl-contextual-navigation__item  ecl-contextual-navigation__item--more"
+    >
+      <button
+        class="ecl-button ecl-button--ghost ecl-contextual-navigation__more"
+        data-ecl-contextual-navigation-more="true"
+        type="submit"
```

Comment:
it shouldn't be a `submit` button ;) (`submit` is the default type)

Reply:
I'm not sure which one it should be, I followed the [current one](https://v2-dev--europa-component-library.netlify.com/ec/components/navigation/contextual-navigation/code/)